massgov/mayflower
# DP-9574 Remove raw filter from 02_molecules/google-map.twig

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Remove raw filter from styleguide/source/_patterns/02-molecules/google-map.twig

Use of the template in Mayflower:
```
 ~/MassIT/mayflower/patternlab/ [feature/DP-9574_molecules-google-map] grep -r 'google-map.twig' styleguide/source
styleguide/source/_patterns/03-organisms/by-author/location-banner.twig:    {% include "@molecules/google-map.twig" %}
styleguide/source/_patterns/03-organisms/by-author/location-listing.twig:          {% include "@molecules/google-map.twig" %}
styleguide/source/_patterns/03-organisms/by-author/mapped-locations.twig:    {% include "@molecules/google-map.twig" %}
styleguide/source/_patterns/05-pages/location-general-content.twig:  {% include "@molecules/google-map.twig" %}
styleguide/source/_patterns/05-pages/location-park-content.twig:  {% include "@molecules/google-map.twig" %}
styleguide/source/assets/js/modules/googleMap.js:      const rawData = ma.googleMapData[i]; // Data object created in @molecules/google-map.twig
 ~/MassIT/mayflower/patternlab/ [feature/DP-9574_molecules-google-map]
```

## Related Issue / Ticket

- [DP-9574 Mayflower: Remove "raw" filter from 02-molecules/google-map.twig](https://jira.mass.gov/browse/DP-9574)

## Steps to Test


1. Go to the Location Listing page and check for any display issue with listing item info.
2. Go to the Location pages and check for any display issue with the banner map.

## Screenshots
Location Listing:
<img width="899" alt="dp-9574_mf-location-listing" src="https://user-images.githubusercontent.com/9633303/43407823-c0df59ae-93ed-11e8-9558-180ffdf3e2af.png">

Location:
<img width="882" alt="dp-9574_mf-location-banner" src="https://user-images.githubusercontent.com/9633303/43408590-bd717f02-93ef-11e8-849a-9553dc4fc051.png">

<img width="760" alt="dp-9574_mf-location-general" src="https://user-images.githubusercontent.com/9633303/43408613-ca16327a-93ef-11e8-9bc5-f016a9270042.png">


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
